modules/codemod/lib/v5/removeButtonEnums.ts

```diff
@@ -14,6 +14,7 @@ const importsToRemove = [
   'ButtonVariant',
   'ButtonIconPosition',
   'ButtonSize',
+  'ButtonIconSize',
```

Comment:
Just double-checking the remove button enums are an intentional addition to the PR. I didn't see anything about this in the PR description.

Reply:
It is an enum that was supposed to be removed, but missed. Thanks for calling this out